### DISCUSSION TITLE
Newly created bundle not revealed

### DISCUSSION THREAD

**Discussion by chregi** [pimcore/pimcore#13900] [General]
I have created a new bundle with the BundleGenerator. The individual files were created as desired from my point of view. However, when I query the bundle list via the console, my bundle is not listed. What do I have to do so that the bundle is listed?

**Comment by dvesh3** [comment]
first step is to register the bundle in `config/bundles.php` https://github.com/pimcore/demo/blob/11.x/config/bundles.php
